weldx/measurement.py

```diff
@@ -207,25 +830,76 @@ def _signal_label(signal):
         )
 
         # draw edge labels
+        edge_labels = {
+            edge: _transformation_label(
+                edge[1], self._graph.edges[edge]["transformation"]
+            )
+            for edge in self._graph.edges
+        }
+
         draw_networkx_edge_labels(graph, positions, edge_labels, ax=axes)
 
         return axes
 
+    def to_tree(self) -> Dict:
+        """Get the content of the measurement chain as dictionary.
 
-@dataclass
-class Measurement:
-    """Simple dataclass implementation for generic measurements."""
+        Returns
+        -------
+        Dict:
+            Content of the measurement chain as dictionary.
 
-    name: str
-    data: Data
-    measurement_chain: MeasurementChain
+        """
+        return dict(
+            name=self._name,
+            data_source=self._source,
+            source_equipment=self._source_equipment,
+            source_signal=build_tree(self._graph, self._source.name),
+        )
+
+    @property
+    def source(self) -> SignalSource:
+        """Return the source of the measurement chain.
+
+        Returns
+        -------
+        SignalSource :
+            The source of the measurement chain
+
+        """
```

Comment:
```suggestion
        """Return the source of the measurement chain."""
```

Reply:
I think we should stick with the full docstring for public methods since it is also used for the documentation.